sdks/builds/llvm.mk

```diff
@@ -123,6 +129,16 @@ TARGETS += llvm-$(1)
 
 endef
 
+# -DLLVM_ENABLE_THREADS=0 is needed because mxe doesn't define std::mutex etc.
+# -DLLVM_BUILD_EXECUTION_ENGINE=Off is needed because it depends on threads
+# -DCROSS_TOOLCHAIN_FLAGS_NATIVE is needed to compile the native tools (tlbgen) using the host compilers
+llvm_win_configure_flags = -DLLVM_ENABLE_THREADS=0 -DCROSS_TOOLCHAIN_FLAGS_NATIVE=-DCMAKE_TOOLCHAIN_FILE=$(LLVM_SRC)/cmake/modules/NATIVE.cmake -DLLVM_BUILD_EXECUTION_ENGINE=Off
+
+llvm_llvmwin64_CMAKE = $(MXE_PREFIX)/bin/x86_64-w64-mingw32.static-cmake
```

Comment:
Construct this variable at https://github.com/mono/mono/pull/8260/files#diff-e19948cc4c89121d1aa1068fd3d60b01R107 with `$(2)-w64-mingw32$$(if $$(filter $$(UNAME),Darwin),.static)`, since you need to treat both Linux and macOS case.